CONFIG.md

```diff
@@ -2834,6 +2835,39 @@ require'lspconfig'.perlls.setup{}
     }
 ```
 
+## php_language_server
+
+https://github.com/felixfbecker/php-language-server
+
+PHP Implementation of the VS Code Language Server Protocol
+
+`php-language-server` can be installed via `composer`:
+```sh
+# To install the language server
+composer require felixfbecker/language-server
+# To parse stubs
+composer run-script --working-dir=vendor/felixfbecker/language-server parse-stubs
+```
+
+To use `php-language-server`, you must define `cmd` when you setup the language server:
+```lua
+require'lspconfig'.sumneko_lua.setup {
```

Comment:
Shouldn't this be php_language_server?

Reply:
No, we use the official name in the docs.  Substitute underscores is just a lua thing.

Reply:
Sorry, I'm not clear. It requires `sumneko_lua` for php server?

Reply:
Nope, it's a typo